reference_system_autoware/test/generate_graph.sh

```diff
@@ -0,0 +1 @@
+psrecord "ros2 run reference_system_autoware $1" --log $2 --plot $3 --duration $4
```

Comment:
I think you forgot the `#!/bin/bash` header.

I would add some lines which check if `psrecord` is installed and provide some hint if not.
```sh
if ! [ -x "$(command -v psrecord)" ]                                         
then                                                                         
    echo Please install psrecord and make it available via the PATH variable.
    echo \# pip install psrecord                                             
    echo \# export PATH=\${PATH}:\${HOME}/.local/bin/                        
    exit -1                                                                  
fi                                                                           
```